custom_components/tuya_local/devices/zigbee_smart_bulb.yaml

```diff
@@ -0,0 +1,89 @@
+name: Zigbee Light Bulb
+products:
+  - id: mja6r5ix
+    name: Zigbee Smart Bulb
```

Comment:
These names that are for a specific product ID should include the branding, not be generic like this.

Also for all device and entity names, you should follow the HA naming convention of capitalising only the first word.

Reply:
there's no brand at all in the product or package. should I remove the "products" section then?